packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -482,32 +495,37 @@ class Auth {
     }
 
     /**
-     * This method stores the TokenResponse object retrived from SLAS, and
+     * This method stores the TokenResponse object retrieved from SLAS, and
      * store the data in storage.
      */
     private handleTokenResponse(res: TokenResponse, isGuest: boolean) {
         this.set('access_token', res.access_token)
         this.set('customer_id', res.customer_id)
         this.set('enc_user_id', res.enc_user_id)
         this.set('expires_in', `${res.expires_in}`)
-        this.set('refresh_token_expires_in', res.refresh_token_expires_in)
         this.set('id_token', res.id_token)
         this.set('idp_access_token', res.idp_access_token)
         this.set('token_type', res.token_type)
         this.set('usid', res.usid)
         this.set('customer_type', isGuest ? 'guest' : 'registered')
 
         const refreshTokenKey = isGuest ? 'refresh_token_guest' : 'refresh_token_registered'
-        const expiresDate = res.refresh_token_expires_in
-            ? this.convertSecondsToDate(res.refresh_token_expires_in)
-            : undefined
+        const fallbackTTL =
+            typeof this.refreshTokenCookieTTL === 'number' &&
+            this.refreshTokenCookieTTL >= 0 &&
+            this.refreshTokenCookieTTL <= ninetyDaysInSeconds
+                ? this.refreshTokenCookieTTL
+                : res.refresh_token_expires_in || ninetyDaysInSeconds
+
+        this.set('refresh_token_expires_in', fallbackTTL.toString())
+        const expiresDate = this.convertSecondsToDate(fallbackTTL)
 
         this.set(refreshTokenKey, res.refresh_token, {
             expires: expiresDate
         })
     }
 
-    async refreshAccessToken() {
+    async refreshAccessToken(accessToken?: string) {
```

Comment:
can we remove the `accessToken` parameter and use the `this.getAccessToken()`? It reduces the complexities of the interface

Reply:
@johnnygreen I'll leave you to answer that. I suppose `this.get('accessToken')` could be used?

Reply:
yeah of course, where should i commit this change now that we have https://github.com/SalesforceCommerceCloud/pwa-kit/pull/2089?